### PR TITLE
Feature: routing

### PR DESCRIPTION
**Pull request by skiffua**
Made routing with 2 pages and apply state

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,30 +1,23 @@
 <template>
   <div id="app">
```
Comment (stanvol) [inline src/App.vue]: Do you need this `div`? You can just `<router-view></router-view>`

File: src/App.vue [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
+<style lang="scss">
+    @import './assets/app';
 
-  #app {
-    font-family: "Avenir", Helvetica, Arial, sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-  }
+#app {
+  font-family: "Avenir", Helvetica, Arial, sans-serif;
```
Comment (stanvol) [inline src/App.vue]: Move this styles to app.scss -> body {...}

File: src/components/Calculator.vue [hunk 3]
```diff
@@ -175,7 +178,8 @@
                     this.result();
 
                     const result = eval(this.calc_screen).toString();
-                    this.history.push(this.calc_screen + '=' + result);
+                    this.pushOperationInState(
+                        '<div class="expression">'+this.calc_screen+'</div><div class="equalSymbol">=</div><div class="expression">'+result+'</div>')
```
Comment (stanvol) [inline src/components/Calculator.vue]: It is bad idea to push html here. All html should be in template, in History page. Push `this.calc_screen` or if you want to style components (firstNumber, operation, secondNumber) you can push object.

File: src/main.js [hunk 4]
```diff
@@ -1,9 +1,15 @@
 import Vue from 'vue'
 import App from './App.vue'
+import VueRouter from "vue-router";
+import router from './routes'
+import "normalize.css"
```
Comment (stanvol) [inline src/main.js]: Why do you add it again here?

File: src/pages/History.vue [hunk 5]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <div>
+        <Header/>
+        <div class="environment-block">
+            <h1>Try to use calculator</h1>
+            <button class="change-calc-view" @click="cleanHistory()">Clean history</button>
```
Comment (stanvol) [inline src/pages/History.vue]: Add prefix `on` to methods that are used in template.

File: src/pages/History.vue [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    export default {
+
+        name: "History",
+
+        components: {
+            Header,
+        },
+
+        computed: {
+          getHistoryFromState(){
```
Comment (stanvol) [inline src/pages/History.vue]: It is better to name `historyList`

File: src/main.js (hunk 4 above)
Comment (skiffua) [reply src/main.js]: it's get back from "rebase". Deleted it

File: src/App.vue (hunk 1 above)
Comment (skiffua) [reply src/App.vue]: ok, I thought that root div 'app' is needed

File: src/components/Calculator.vue (hunk 3 above)
Comment (skiffua) [reply src/components/Calculator.vue]: rewrited

**Pull request merged**